### DISCUSSION TITLE
pcbソケット部 ランド剥がれの修復方法について

### DISCUSSION THREAD

**Discussion by jun3366** [Yowkees/keyball#198] [Q&A]
keyball39を右手トラックボールにて使わせていただいています。
左手の「G」のボタンが下記のような症状になりました。このようになった時の対策方法をご教授いただきたいです。
①他のキーが反応するような強さで押しても反応しない
②強く押すと反応するが、1〜3回分入力されてしまう。
③ソケットを再はんだをしキーを押すと20分ほど正常に入力されますが、①,②に戻ってしまいます。

また、予備をソケットがあり、交換すれば解決するのではと思い、ソケットを取り外しました。
その時にパターンが剥がれてしまい、再はんだするも反応しなくなりました。
ネットで調べたところやすりで削り、銅を露出させ再はんだを行うと記載がありました。
上記対応する場合、どこをヤスリで削ればよいか、違う対応で修復する場合、修復方法をご教授いただきたいです。

>

**Comment by Yowkees** [comment]
ランド剥がれ了解しました。
下記画像の赤点２点をエナメル線やゴム被覆導線等を使用して
接続してください。これでキーが復活するはずです。

以上、お手数ですがご対応よろしくお願いいたします。

![39FIX](https://user-images.githubusercontent.com/84209772/220020506-dc2b9f57-a813-41b9-90a0-2274bf079983.jpg)
